File: fixtures/python_output/multiple_d_post.py
import requests

data = [
  ('version', '1.2'),
  ('auth_user', 'fdgxf'),
  ('auth_pwd', 'oxfdscds'),
  ('json_data', '{ "operation": "core/get", "class": "Software", "key": "key" }'),
]

response = requests.post('https://cmdb.litop.local/webservices/rest.php', data=data)

File: fixtures/r_output/multiple_d_post.R
require(httr)

data = list(
  `version` = '1.2',
  `auth_user` = 'fdgxf',
  `auth_pwd` = 'oxfdscds',
  `json_data` = '{ "operation": "core/get", "class": "Software", "key": "key" }'
)

res <- httr::POST(url = 'https://cmdb.litop.local/webservices/rest.php', body = data)
